src/containers/ContentContainer.jsx

```diff
@@ -23,6 +24,26 @@ const ContentContainer = function ({ html }) {
   // is always highlighted even after updates
   useEffect(() => {
     Prism.highlightAll();
+
+    const pres = document.getElementsByTagName("pre");
+
+    if (pres !== null) {
+      for (let codeSnippet of pres) {
+        if (codeSnippet.className === ' language-js' || codeSnippet.className === ' language-jsx') {
+          codeSnippet.innerHTML = `<div class="copy"><i class="fas fa-copy"></i></div><code class="${codeSnippet.className}">${codeSnippet.innerHTML}</code>`;
+        }
+      }
+      const clipboard = new Clipboard('.copy', {
+        target: trigger => {
```

Comment:
This line could be better `target: ({ nextElementSibling }) => nextElementSibling`

Reply:
Will change!